### PR TITLE
update schema docs intro

### PR DESCRIPTION
**Pull request by danabauer**
this is a first pass at updating the schema docs intro page and giving users action items. See https://dfhx9f55j8eg5.cloudfront.net/pr/105

### CODE REVIEW

**Comment by danabauer** [comment]
Do we want to keep this under key specifications?
` Property names use camelCase as described in the [Google JSON Style Guide](https://google.github.io/styleguide/jsoncstyleguide.xml).`

File: docusaurus/docs/index.mdx [hunk 1]
```diff
@@ -18,12 +26,13 @@ less common use cases ("the periphery") via extensibility.
...
+
 ## Key Specifications
-1. The Overture Data Schema is documented in  [JSON Schema](https://json-schema.org/), a widely used, machine-readable schema declaration language for JSON documents.
-2. We use GeoJSON as a mental model. The final format for Overture deliveries is still to be defined.
-3. Property names use camelCase as described in the [Google JSON Style Guide](https://google.github.io/styleguide/jsoncstyleguide.xml).
-4. Feature Geometries follow the [OGC Simple Feature Access specifications](https://www.ogc.org/standard/sfa/).
-5. Measurements use SI units (e.g. meters for building heights) and local units for regulations and restrictions (e.g. speed limits in mph in the US and km/h in Germany).
+- The Overture Data Schema is documented in [JSON Schema](https://json-schema.org/), a widely used, machine-readable schema declaration language for JSON documents.
+- We use GeoJSON as a mental model. We aim to deliver our data in the most developer-friendly formats. 
+- Property names use camelCase as described in the [Google JSON Style Guide](https://google.github.io/styleguide/jsoncstyleguide.xml).
```
Comment (danabauer) [inline docusaurus/docs/index.mdx:33]: do we want to keep this? or change it?

**Comment by danabauer** [comment]
I'm realizing now that we'll have to reconcile this with #102

**Comment by jenningsanderson** [comment]
> @jenningsanderson I'm realizing now that we'll have to reconcile this with #102

Love it — not a problem, I can resolve any ugliness here.

**Comment by jenningsanderson** [comment]
Merged this in & working with the new names-documentation branch to get this ready for Feb. release.

**Pull request merged**